eml/components/Ai/FeedbackButtons.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { View, TouchableOpacity } from 'react-native';
+import { Icon } from '@rneui/themed';
+import { sendFeedbackToBackend } from '../../api/api';
+import PropTypes from 'prop-types';
+
+const FeedbackButtons = ({ aiText, userText }) => {
+	const [selectedFeedback, setSelectedFeedback] = useState(null);
+
+	const handleFeedback = async (type) => {
+		const feedback = type === 'thumbsUp'; // thumbsUp -> true, thumbsDown -> false
```

Comment:
I would sugest that you change this to either:
- True / False
- Enum, thumbsUp / ThumbsDown
This will help the future developers, as it is easier to understand